configs/configschema/schema.go

```diff
@@ -72,7 +77,7 @@ type Attribute struct {
 	Deprecated bool
 }
 
-// NestedBlock represents the embedding of one block within another.
+// NestedType represents the embedding of a block-like object within an Attriubte.
```

Comment:
Shouldn't this still be NestedBlock? The struct property name changed, but the struct name did not, it looks like?

Reply:
(Also: typo, Attriubte)

Reply:
fixed and fixed, thank you